tests/tensor/rewriting/test_linalg.py

```diff
@@ -390,3 +392,32 @@ def test_local_lift_through_linalg(constructor, f_op, f, g_op, g):
     test_vals = [x @ np.swapaxes(x, -1, -2) for x in test_vals]
 
     np.testing.assert_allclose(f1(*test_vals), f2(*test_vals), atol=1e-8)
+
+
+def test_local_svd_uv_simplify():
+    a = matrix("a")
+    s_1 = svd(a, full_matrices=False, compute_uv=False)
+    _, s_2, _ = svd(a, full_matrices=False, compute_uv=True)
+    # full_matrices = True is not supported for grad of svd
+    gs = pt.grad(pt.sum(s_1), a)
```

Comment:
Explain that `grad` introduces the svd with compute_uv=True.

Assert you have two SVDs in the original graph of gs, with and without compute_uv. This will make the test more readable

Reply:
Hi @ricardoV94, this may be a stupid question, but how can I access the graph of `gs`?

Reply:
Not a stupid question. For yourself you can always call pytensor.dprint to get a text representation of the graph. To programmatically transverse a graph you can use utilities in pytensor.graph.basic, like ancestors which yields the nodes above a certain variable.

This page in the docs may be useful: https://pytensor.readthedocs.io/en/latest/extending/graphstructures.html